test/test_helpers.bash

```diff
@@ -4,6 +4,10 @@
 # tests fail when it is set to something other than the temp dir.
 unset ASDF_DIR
 
+# Set an agnostic Git configuration directory to prevent personal
+# configuration from interfering with the tests
+export GIT_CONFIG_GLOBAL=/dev/null
```

Comment:
Is there a way to set this with a Bats function so it is only applied during the Bats test suite lifecycle?

Reply:
Bats added `setup_suite` several months ago - I can add it in there. Shall I move `ASDF_DIR` there as well?

Reply:
Yes please, we would like to utilise Bats built-in setup & teardown capabilities where possible. Would be good to refactor the other tests in a similar way in a follow-up PR.

Reply:
Done that :+1: 

> Would be good to refactor the other tests in a similar way in a follow-up PR.

Am I correct to understand that this means moving `load test_helpers` in`setup_suite()` as well? I'm not sure how else to refactor the other tests in a similar way

Reply:
`setup_suite()` runs once before all tests. The other `setup` and `teardown` run before and after each test.

So, refactoring of these other tests might not be necessary. I think it makes sense in this instance to remove the global `unset ASDF_DIR` in the `test/test_helpers.bash` file.